canary/ClientApp/src/components/tests/Connectathon.js

```diff
@@ -159,14 +160,16 @@ export class Connectathon extends Component {
                 <Header as="h2" dividing id="step-1">
                   <Icon name="flag" />
                   <Header.Content>
-                    Step 1: Select State
+                    Step 1: Set Certificate Number and Select Jurisdiction
                     <Header.Subheader>
-                      Select the state which you are generating a message from.
+                      Specify the certificate number to be used in the record and select the jurisdiction which you are generating a record from.
                     </Header.Subheader>
                   </Header.Content>
                 </Header>
-                <div className="p-b-15" />
-                <Dropdown placeholder='Select State' search selection fluid onChange={this.fetchTest} options={stateOptions} />
+                <h4>Certificate Number</h4>
+                <Input value={this.state.certificateNumber} placeholder='Enter Certificate Number' fluid onChange={this.updateCertificateNumber} />
```

Comment:
It is currently impossible to clear the certificate number field completely, and instead always requires 1 character in it. Slightly strange behavior and may be slightly confusing for users trying to delete the whole certificate in order to type a new certificate number in.

This is fixable by replacing `<Input value` with `<input defaultValue`.

Reply:
Done!